rules/CodeQuality/Rector/FunctionLike/SimplifyUselessVariableRector.php

```diff
@@ -148,6 +185,10 @@ private function shouldSkipStmt(Return_ $return, Stmt $previousStmt): bool
             return true;
         }
 
+        if ($this->skipConcat && $previousNode instanceof AssignOp\Concat) {
```

Comment:
It not always concat, can be other operator, eg : `*`, see https://getrector.com/demo/3e6c0b33-9692-44d9-8282-7cdbbf4607be

Reply:
That's why you suggested `only_direct_assign`. Got it. Should I change it to that? And exclude any AssignOp?

Reply:
Yes, I personally prefer that

Reply:
Done